assets/styles/mobile-about.css

```diff
@@ -0,0 +1,294 @@
+}
+
+@media screen and (max-width: 768px) {
+  .top-nav,
+  .logo-nav {
+    display: none;
+  }
+
+  .nav-mobile-version {
+    display: flex;
+    padding: 20px;
+  }
+
+  .humberger-menu {
+    margin-right: auto;
+    cursor: pointer;
+    z-index: 100;
+    margin-left: 20px;
+    list-style: none;
+  }
+
+  .menu-link-bar {
+    width: 25px;
+    height: 5px;
+    margin: 5px 0;
+    background-color: rgb(68, 66, 66);
+    transition: 0.5s;
+  }
+
+  .humberger-menu.change .menu-link-bar.span-one {
+    transform: translate(0, 11px) rotate(-45deg);
+    z-index: 3;
+    margin-top: 20%;
+    list-style: none;
+    background-color: #fdfdf7;
+  }
+
+  .humberger-menu.change .menu-link-bar.span-two {
+    opacity: 0;
+  }
+
+  .humberger-menu.change .menu-link-bar.span-three {
+    transform: translate(0, -9px) rotate(45deg);
+    z-index: 3;
+    background-color: #fdfdf7;
+  }
+
+  .bottom-menu-mobile {
+    display: none;
+  }
+
+  .show-menu {
+    display: block;
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    z-index: 99;
+    top: 0;
+    left: 0;
+    background-color: rgb(48, 45, 45);
+    background-color: rgba(51, 46, 46, 0.9);
+    overflow-x: hidden;
+    transition: 0.5s;
+    backdrop-filter: blur(2px);
+  }
+
+  .main-nav-links {
+    display: flex;
+    justify-content: center;
+    gap: 30px;
+    padding: 20px 0;
+    margin-top: -80px;
+    margin-left: 200px;
+  }
+
+  .menu-link-bottom {
+    color: #ebe6e6;
+    font-size: 5vw;
+    font-weight: 70;
+    padding: 10%;
+    list-style: none;
+    margin-left: 20px;
+  }
+
+  .bottom-nav-links {
+    margin-top: 70px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    list-style: none;
+  }
+
+  .block-tiles li {
+    flex: 0 0 100%;
+    line-height: 1;
+  }
+
+  .about-hero {
+    background-image: url(../images/mobile_intro_bg.png);
+    background-repeat: no-repeat;
+  }
+
+  .about-hero > h1 {
+    font-size: 20px;
+    font-weight: 700;
+    text-align: center;
+    color: orangered;
+    padding: 30px;
+  }
+
+  .about-hero > h2 {
+    font-size: 30px;
+    font-weight: 990;
+    text-align: center;
+    color: orangered;
+  }
+
+  .intro-about {
+    align-items: center;
+    padding: 50px;
+    margin-right: 10px;
+    font-size: 20px;
+    color: #272a31;
+  }
+
+  .email-contact {
+    margin-top: -20px;
+    color: #272a31;
+    width: 350px;
+    margin-left: 15%;
+  }
+
+  .hiddekel-email {
+    width: 100px;
+    font-size: 20px;
+    margin-left: 15%;
+    padding: 15px;
+    color: #272a31;
+  }
+
+  .hiddekel-logo {
+    height: 500px;
+  }
+
+  .logo-text {
+    font-size: 30px;
+    font-weight: 990;
+    text-align: center;
+    color: #575b63;
+    padding: 30px;
+  }
+
+  .line {
+    width: 45px;
+    height: 3px;
+    border-radius: 50px;
+    background-color: #e03623;
+    margin: 0 auto;
+    margin-top: -13px;
+  }
+
+  .logo-caption {
+    width: 500px;
+    padding: 50px;
+  }
```

Comment:
[optional] please see there is a horizontal overflow in the mobile view please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210617306-0fed67e2-9229-492a-b6ae-2e72d6affb91.png)
I think it is because you have fixed the width of your paragraph for 500px

Reply:
- Sure I removed the width.